image/cli-base/install/install-aws.sh

```diff
@@ -33,6 +33,10 @@ if [[ "$TARGET_PLATFORM" == "s390x" ]]; then
   echo "aws cli not supported in s390x"
   exit 0
 fi
+if [[ "$TARGET_PLATFORM" == "ppc64le" ]]; then
```

Comment:
Needs a readme update to state that aws is not available on power

Reply:
Done